jib-core/src/main/java/com/google/cloud/tools/jib/builder/BuildConfiguration.java

```diff
@@ -200,7 +190,7 @@ public BuildConfiguration build() {
               ImmutableList.copyOf(javaArguments),
               ImmutableList.copyOf(jvmFlags),
               ImmutableMap.copyOf(environmentMap),
-              expandPortRanges(exposedPorts),
+              ExposedPorts.expandPortRanges(exposedPorts, buildLogger),
```

Comment:
We should try to keep `BuildConfiguration` free of user input parsing. `BuildConfiguration` should really be passed in a list of `PortWithProtocol` objects.

Reply:
Also, in general, we should not have other parts of jib-core depend on frontend classes.